src/cubic_loader/pipeline.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from cubic_loader.utils.aws import check_for_parallel_tasks
+from cubic_loader.utils.logger import ProcessLogger
+from cubic_loader.utils.postgres import alembic_upgrade_to_head
+from cubic_loader.utils.postgres import DatabaseManager
+from cubic_loader.utils.runtime import validate_environment
+from cubic_loader.utils.remote_locations import ODS_SCHEMA
+
+
+from cubic_loader.dmap.api_copy_job import run_api_copy
+from cubic_loader.dmap.api_job_list import produce_job_list
+
+from cubic_loader.qlik.ods_tables import CUBIC_ODS_TABLES
+from cubic_loader.qlik.ods_qlik import CubicODSQlik
+
+
+def start_dmap() -> None:
+    """
+    Upgrade DB and run DMAP api jobs.
+
+    Jobs are run without any re-try logic
+
+    For any API URL, if a result loading throws, no more results are processed
+    for that URL. During the next processing loop, the result that threw should
+    be re-processed based on filtering and ordering of `last_updated` result
+    field.
+    """
```

Comment:
DMAP process should run same as before with no changes. 

Any changes to DMAP files should only be re-naming, changing imports or modifying formatting.